[2023-03-17 14:13:17,604] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_ingestion_v1.0.create_log_task_table manual__2023-03-17T14:10:46.193794+00:00 [queued]>
[2023-03-17 14:13:17,699] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_ingestion_v1.0.create_log_task_table manual__2023-03-17T14:10:46.193794+00:00 [queued]>
[2023-03-17 14:13:17,704] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-17 14:13:17,710] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2023-03-17 14:13:17,813] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-17 14:13:17,924] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_log_task_table> on 2023-03-17 14:10:46.193794+00:00
[2023-03-17 14:13:17,945] {standard_task_runner.py:52} INFO - Started process 1934 to run task
[2023-03-17 14:13:17,985] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_ingestion_v1.0', 'create_log_task_table', 'manual__2023-03-17T14:10:46.193794+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpmoot0uje', '--error-file', '/tmp/tmpd0f8qbli']
[2023-03-17 14:13:17,994] {standard_task_runner.py:77} INFO - Job 98: Subtask create_log_task_table
[2023-03-17 14:13:18,260] {logging_mixin.py:109} INFO - Running <TaskInstance: local_ingestion_v1.0.create_log_task_table manual__2023-03-17T14:10:46.193794+00:00 [running]> on host 8920ab98807f
[2023-03-17 14:13:18,951] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_ingestion_v1.0
AIRFLOW_CTX_TASK_ID=create_log_task_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-17T14:10:46.193794+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-17T14:10:46.193794+00:00
[2023-03-17 14:13:19,118] {base.py:79} INFO - Using connection to: id: postgress_localhost. Host: host.docker.internal, Port: 5432, Schema: wine_quality, Login: ***, Password: ***, extra: {}
[2023-03-17 14:13:19,240] {dbapi.py:225} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2023-03-17 14:13:19,663] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_ingestion_v1.0, task_id=create_log_task_table, execution_date=20230317T141046, start_date=20230317T141317, end_date=20230317T141319
[2023-03-17 14:13:19,871] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-17 14:13:20,133] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
